From c5c60ded6dbb6256640b0308a45c42da8c7fe071 Mon Sep 17 00:00:00 2001
From: akr <akr@b2dd03c8-39d4-4d8f-98ff-823fe69b080e>
Date: Fri, 22 Apr 2016 12:20:06 +0000
Subject: [PATCH] * test/ruby/test_time_tz.rb: Tests depends on Europe/Moscow
 removed   to avoid test failures due to the tzdata change.  
 https://github.com/eggert/tz/commit/8ee11a301cf173afb0c76e0315b9f9ec8ebb9d95 
  Found by naruse.

git-svn-id: svn+ssh://ci.ruby-lang.org/ruby/trunk@54706 b2dd03c8-39d4-4d8f-98ff-823fe69b080e
---
 ChangeLog                 |  7 +++++++
 test/ruby/test_time_tz.rb | 13 -------------
 2 files changed, 7 insertions(+), 13 deletions(-)

diff --git a/ChangeLog b/ChangeLog
index 3b96a9b..965f296 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,3 +1,10 @@
+Fri Apr 22 21:00:44 2016  Tanaka Akira  <akr@fsij.org>
+
+  * test/ruby/test_time_tz.rb: Tests depends on Europe/Moscow removed
+    to avoid test failures due to the tzdata change.
+    https://github.com/eggert/tz/commit/8ee11a301cf173afb0c76e0315b9f9ec8ebb9d95
+    Found by naruse.
+
 Wed Dec 16 21:16:55 2015  CHIKANAGA Tomoyuki  <nagachika@ruby-lang.org>
 
 	* ext/fiddle/handle.c: check tainted string arguments.
diff --git a/test/ruby/test_time_tz.rb b/test/ruby/test_time_tz.rb
index f5d4690..b40b1ac 100644
--- a/test/ruby/test_time_tz.rb
+++ b/test/ruby/test_time_tz.rb
@@ -161,13 +161,6 @@ def test_europe_lisbon
     }
   end if has_lisbon_tz
 
-  def test_europe_moscow
-    with_tz(tz="Europe/Moscow") {
-      assert_time_constructor(tz, "1992-03-29 00:00:00 +0400", :local, [1992,3,28,23,0,0])
-      assert_time_constructor(tz, "1992-03-29 00:59:59 +0400", :local, [1992,3,28,23,59,59])
-    }
-  end
-
   def test_pacific_kiritimati
     with_tz(tz="Pacific/Kiritimati") {
       assert_time_constructor(tz, "1994-12-31 23:59:59 -1000", :local, [1994,12,31,23,59,59])
@@ -346,12 +339,6 @@ def self.gen_zdump_test(data)
 Europe/London  Sun Aug 10 01:00:00 1947 UTC = Sun Aug 10 02:00:00 1947 BST isdst=1 gmtoff=3600
 Europe/London  Sun Nov  2 01:59:59 1947 UTC = Sun Nov  2 02:59:59 1947 BST isdst=1 gmtoff=3600
 Europe/London  Sun Nov  2 02:00:00 1947 UTC = Sun Nov  2 02:00:00 1947 GMT isdst=0 gmtoff=0
-Europe/Moscow  Sat Jan 18 23:59:59 1992 UTC = Sun Jan 19 01:59:59 1992 MSK isdst=0 gmtoff=7200
-Europe/Moscow  Sun Jan 19 00:00:00 1992 UTC = Sun Jan 19 03:00:00 1992 MSK isdst=0 gmtoff=10800
-Europe/Moscow  Sat Mar 28 19:59:59 1992 UTC = Sat Mar 28 22:59:59 1992 MSK isdst=0 gmtoff=10800
-Europe/Moscow  Sat Mar 28 20:00:00 1992 UTC = Sun Mar 29 00:00:00 1992 MSD isdst=1 gmtoff=14400
-Europe/Moscow  Sat Sep 26 18:59:59 1992 UTC = Sat Sep 26 22:59:59 1992 MSD isdst=1 gmtoff=14400
-Europe/Moscow  Sat Sep 26 19:00:00 1992 UTC = Sat Sep 26 22:00:00 1992 MSK isdst=0 gmtoff=10800
 Pacific/Kiritimati  Sun Jan  1 09:59:59 1995 UTC = Sat Dec 31 23:59:59 1994 LINT isdst=0 gmtoff=-36000
 Pacific/Kiritimati  Sun Jan  1 10:00:00 1995 UTC = Mon Jan  2 00:00:00 1995 LINT isdst=0 gmtoff=50400
 End
